src/Core/View.js

```diff
@@ -278,4 +279,49 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * @typedef {object} FrameRequester
+ * @property {Function(dt, firtLoop)} update - Method that will be called each
```

Comment:
> What do you think of my name proposal?

I agree, ameding now.

> The usage in FPContols could be moved in MainLoop:

I don't think so, because now dt is the *real* interval between 2 requestAnimationFrame. So it's never undefined any more. I think it's less magic and more useful this way.